tests/cpp/test_host_ir_jit.cpp

```diff
@@ -69,6 +69,73 @@ TEST_F(HostIrJitTest, HostIrContainer) {
   EXPECT_EQ(jit.outputs().size(), num_inputs);
 }
 
+TEST_F(HostIrJitTest, ConstantSizedTensorAllocate) {
```

Comment:
I'd probably merge this and the deallocation test into one and check for memory usage as you did in the other test. How does that sound to you?